client/src/InfoBase/App.js

```diff
@@ -125,6 +127,30 @@ export class App extends React.Component {
 
     ensure_linked_stylesheets_load();
   }
+
+  state = {
+    showNotification: false,
+    message: "",
+  };
+
+  hideNotification = () => {
+    this.setState({ showNotification: false });
+  };
+
+  componentDidMount() {
+    axios
```

Comment:
Is axios superior than built in `fetch()`?

Reply:
I chose it because: 1. It's primarily what I use haha and 2. I saw it being used in the email_backend by someone else.

A little research says that axios is more compatible with older browsers and auto stringifies data. Should I use fetch instead?

Reply:
I think it's fine!